### DISCUSSION TITLE
Tooltips on right of the screen cannot be seen

### DISCUSSION THREAD

**Discussion by Cromwell1963** [israel-dryer/ttkbootstrap#422] [Q&A]
Hi, when I put tooltips on items to the right of the screen, the tooltip is displayed but almost all of it is off screen, Is there any way to ensure tooltips will be on screen? thanks

**Comment by israel-dryer** [comment]
This should be fixed in the upcoming 2.0 release. Tooltips are now automatically repositioned when they would extend beyond the visible screen area, including near the right edge.

Thanks for reporting it.
